web/run.py

```diff
@@ -0,0 +1,25 @@
+"""
+Runs the web interface version of Chemprop.
+This allows for training and predicting in a web browser.
+"""
+
+import os
+from app import app, db
+from argparse import ArgumentParser
+
+if __name__ == "__main__":
+    parser = ArgumentParser()
+    parser.add_argument('--host', type=str, default='127.0.0.1', help='Host IP address')
+    parser.add_argument('--port', type=int, default=5000, help='Port')
+    parser.add_argument('--debug', action='store_true', default=False, help='Whether to run in debug mode')
+    parser.add_argument('--initdb', action='store_true', default=False, help='Initialize Database')
+    args = parser.parse_args()
+
+    db.init_app(app)
+
+    if args.initdb or not os.path.isfile('chemprop.sqlite3'):
```

Comment:
Any chance you could put `'chemprop.sqlite3'` in a global constant so it's easier to change in the future if we need to?